e2e/cypress/integration/channel_sidebar/dm_gm_behaviour_spec.js

```diff
@@ -247,4 +247,120 @@ describe('DM category', () => {
             });
         });
     });
+
+    it('MM-T2003 Number of direct messages to show', () => {
+        const receivingUser = testUser;
+
+        // * Verify that we can see the sidebar
+        cy.get('#headerTeamName').should('be.visible');
+
+        // # Collapse the DM category (so that we can check all unread DMs quickly without the sidebar scrolling beind an issue)
+        cy.get('button.SidebarChannelGroupHeader_groupButton:contains(DIRECT MESSAGES)').should('be.visible').click();
+
+        // # Create 41 DMs (ie. one over the max displayable read limit)
+        for (let i = 0; i < 41; i++) {
+            // # Create a new user to have a DM with
+            cy.apiCreateUser().then(({user}) => {
```

Comment:
Does it make sense to create 41 new users here? Or should I just assume that 41 users exist already?